### PR TITLE
Include transactional queries

### PR DESCRIPTION
**Pull request by sabetech**
In this pull request:
- Included data modification queries
- Performed queries with transactions

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -10,3 +10,26 @@ VALUES ('Pikachu', '2021-01-07', 1,false, 15.04);
...
+VALUES ('Angemon', '2005-06-12', 1, true, -45);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Boarmon', '2005-06-07', 7, true, -20.4);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Blossom', '1998-10-13', 3, true, -17);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Ditto', '2022-05-14', 4, true, -22);
```
Comment (AmaduKamara) [inline data.sql]: - [x] The weight_kg for these two animals does not have the `-` sign in front of them. Kindly remove the negative(-) sign in front of the numbers. It should be 17 and 22 respectively. Changing them will save you in the long run with queries relating to the kgs. 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @sabetech,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: data.sql (hunk 1 above)
Comment (sabetech) [reply data.sql]: The issue has been fixed. Thanks for pointing that out.

**Review by mateo951 — APPROVED** [review]
### Hi @sabetech, 

Well done making the changes requested by the previous reviewer 😬
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔥
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**